ci/create_initial_account.py

```diff
@@ -19,6 +19,14 @@ async def copy_identity_from_default(hail_credentials_secret_name: str) -> str:
 
     secret = await k8s_client.read_namespaced_secret(hail_credentials_secret_name, 'default')
 
+    try:
+        await k8s_client.delete_namespaced_secret(hail_credentials_secret_name, NAMESPACE)
```

Comment:
what's up with this?

Reply:
Because I was doing this in my namespace that already had a `dgoldste-gsa-key` secret but not a user in the database, the `create_namespaced_secret` failed. I needed a way to force create a secret even if one already existed and this is how we do it in auth/driver.py